tests/db_engine_specs/hive_tests.py

```diff
@@ -233,4 +233,4 @@ def test_get_create_table_stmt() -> None:
                 STORED AS TEXTFILE LOCATION :location
                 tblproperties ('skip.header.line.count'=':header_line_count')""",
         {"delim": ",", "location": "s3a://directory/table", "header_line_count": "101"},
-    )
+    )
```

Comment:
line change missing here

Reply:
Didn't catch that, do you want me to add a new line at the end?

Reply:
It seems the last line change was removed in this PR (the last line should always end in a line change). So just adding that should resolve the linting problem.

Reply:
I didn't add anything here but took whatever is there in the master branch : https://github.com/apache/incubator-superset/blob/master/tests/db_engine_specs/hive_tests.py

 Please do let me know what I need to modify, I'll make the same

Reply:
@villebro Added the new line